common/src/main/java/org/wso2/testgrid/common/TestScenario.java

```diff
@@ -75,6 +75,9 @@
     @Column(name = "is_pre_script_success")
     private boolean isPreScriptSuccessful = false;
 
+    @Column(name = "configChangeSet_name")
```

Comment:
Why mix camelCase and underscore?

Reply:
I will change it to "config_change_set_name". Any better suggestion?